src/server/pfs/server/api_server_v2.go

```diff
@@ -314,3 +318,26 @@ func (a *apiServerV2) DeleteAll(ctx context.Context, request *types.Empty) (resp
 func (a *apiServerV2) BuildCommit(ctx context.Context, request *pfs.BuildCommitRequest) (response *pfs.Commit, retErr error) {
 	return nil, errors.New("v2 does not implement BuildCommit")
 }
+
+func (a *apiServerV2) CreateTempFileSet(server pfs.API_CreateTempFileSetServer) error {
+	fsID, expiresAt, err := a.driver.createTempFileSet(server)
+	if err != nil {
+		return err
+	}
+	return server.SendAndClose(&pfs.CreateTempFileSetResponse{
+		FilesetId: fsID,
+		ExpiresAt: expiresAt.Unix(),
+	})
+}
+
+func (a *apiServerV2) RenewTempFileSet(ctx context.Context, req *pfs.RenewTempFileSetRequest) (*pfs.CreateTempFileSetResponse, error) {
+	ttl := time.Duration(req.ExtendSeconds) * time.Second
+	t, err := a.driver.renewTempFileSet(ctx, req.FilesetId, ttl)
+	if err != nil {
+		return nil, err
+	}
+	return &pfs.CreateTempFileSetResponse{
```

Comment:
Is this information useful to return? If so, it seems like it should be in a different message type because this is not the `CreateTempFileSet` rpc.